.github/workflows/continuous-integration.yml

```diff
@@ -1232,3 +1232,47 @@ jobs:
           channel_id: ${{ env.VETS_WEBSITE_CHANNEL_ID }}
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+  # Please keep steps-evaluation at the end of the workflow as it's meant to evaluate the entirety of the workflow steps that have run.
+  steps_evaluation:
+    runs-on: ubuntu-latest
+    name: Steps Evaluation
+    needs:
+      [
+        'build',
+        'security-audit',
+        'linting',
+        'cypress-tests',
+        'testing-reports-cypress',
+        'archive',
```

Comment:
The downside of having this job need `archive` is that it'll add an extra minute to the CI workflow since the workflow will need to wait for it to finish. This will only affect branches that aren't `main` though, since `main` would run the `Deploy` job in parallel with this at the end of the workflow.

Since `archive` is relatively quick, I'm wondering if it would be better to swap `archive` with `unit-tests` since that job has more interesting metrics. This would also avoid adding an extra minute at the end of the workflow for non-`main` branches. It's not a big deal at the moment since it's only a minute being added. Just something to be cautious of if the duration of this job increases in the future

Reply:
I totally get the concern about adding an extra minute. I think, in general, we'd like to keep track of even if the archive steps are failing to some degree. I wonder if the better solution is to have this kick off using a `workflow_dispatch` trigger after CI/E2E workflows finish. Maybe that's a refactor I can work on today.

Reply:
Curt, I like the idea of a refactor to kick off a separate workflow. That should eliminate the concern around workflow completion times.

Reply:
I like that approach of refactoring this into a separate workflow. That would also ensure this runs at the end of the workflow runs so you wouldn't need to check individual jobs. We also won't have to worry about the duration of this job increasing in the future since it'll be in it's own workflow